chart/openfaas/templates/queueworker-dep.yaml

```diff
@@ -60,6 +58,10 @@ spec:
         - name: faas_function_suffix
           value: ".{{ .Values.functionNamespace }}.svc.{{ .Values.kubernetesDNSDomain }}"
         {{- end }}
+        {{- if .Values.queueWorker.maxInflight }}
```

Comment:
Given the implementation, this if statement isn't really needed, any integer value is "valid" and the queue worker will handle values <= 1 correctly, we might as well simplify the chart here by removing the if block

Reply:
I suspect it wouldn't handle 0 and < 1

Reply:
It does infact handle that case, i checked it already

Reply:
When it only starts the parallel goroutines when the value is greater than one, otherwise it is the default single handler https://github.com/openfaas/nats-queue-worker/pull/95/files#diff-e871404fd69cab40521d6b2178847cf7R77

Reply:
line 75 passes it to stan. Unfortunately that is not as forgiving. ... 
If it keeps the helm portion simpler I can move that line into the if...

Reply:
just let me know what you want me to do about that.

Reply:
@alexellis , since you merged I'll fix it in the queue worker